clients/drcachesim/simulator/tlb.h

```diff
@@ -46,21 +49,67 @@ namespace drmemtrace {
 
 class tlb_t : public caching_device_t {
 public:
+    tlb_t(const std::string &name = "tlb");
+
     void
     request(const memref_t &memref) override;
 
-    // TODO i#4816: The addition of the pid as a lookup parameter beyond just the tag
-    // needs to be imposed on the parent methods invalidate(), contains_tag(), and
-    // propagate_eviction() by overriding them.
+    // TODO i#4816: The addition of the pid as a lookup parameter beyond just the
+    // tag needs to be imposed on the parent methods invalidate(), contains_tag(),
+    // and propagate_eviction() by overriding them.
```

Comment:
This didn't need to change, looks like.  When I re-clang-format this file in my editor I don't see "tag" moved to the next line: it only hits column 85.  Are you sure you have your settings configured for this project?  We have 90-char lines.  This seems like it should be reverted.